src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    public String value;
+    private boolean isPrivate;
+
+    public Contact(boolean isPrivate) {
+        this.isPrivate = isPrivate;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
In general, `equals()` and `hashCode()` methods should be overridden together. Find out why!